common/xfer/plugins.go

```diff
@@ -0,0 +1,216 @@
+package xfer
```

Comment:
Not sure. I moved it in here to match `common/xfer/controls.go`. It's also used in the registry and reporter. Slightly unclear where it should go.